Scripts/README.md

```diff
@@ -108,9 +94,52 @@ some zone a zero value line instead of a missing line must be used.
 
 ### `ITERATION_COUNT`
 
-Number of demand model iterations to run
+Maximum number of demand model iterations to run
 (each using re-calculated impedance from traffic and transit assignment).
 
-### `USE_FIXED_TRANSIT_COST`
+### `MAX_GAP`
+
+Convergence criterion: Car work matrix maximum change between iterations.
+
+### `REL_GAP`
+
+Convergence criterion: Car work matrix relative change between iterations.
+
+### `OPTIONAL_FLAGS`
+
+These should not be used when running model system from command line!
+Instead `helmet.py` flag parameters should be used.
+These can be set if model system is run from UI, to set parameters that cannot be set in UI.
```

Comment:
What do you mean by these instructions?

There are three ways to run the model with parameters, right?
1. UI
2. CLI with parameters in the command line call e.g. `python helmet.py --scenario-name awesomescenario`
3. CLI with parameters in the `dev-config.json` when the command line call is just `python helmet.py`

Should this say that optional flags should not be used in `dev-config.json` so that they do not override UI parameters?

Reply:
Oops, I did not realize we were at `dev-config.json` manual. 😄 I'm going to leave this here anyway but it is not that relevant anymore.